primitives/sr-arithmetic/fuzzer/Cargo.lock

```diff
@@ -308,7 +308,7 @@ dependencies = [
 ]
 
 [[package]]
-name = "sr-std"
+name = "std"
```

Comment:
have you tested whether the fuzzer still works?

Reply:
Looks like this is not in CI

Reply:
Already fixed the cargo.lock file, but the ci code still needs to add